### PR TITLE
Remove deprecated properties alert and error

### LINKED ISSUE #7015: Deprecated Features entfernen
Umfang: über 60 Codestellen.

- [x] Komponenten: u.a. Indented Text, Button Group
- [x] Properties: u.a. `_alert`, `_error`
- [x] Methoden
- [x] Breaking Changes dokumentieren
- [x] Migrationen müssen geschrieben werden.
---

- [x] Docs

PRs werden aufgeteilt:

- [x] https://github.com/public-ui/kolibri/pull/7205
- [x] https://github.com/public-ui/kolibri/pull/7213
- [x] https://github.com/public-ui/kolibri/pull/7214
- [x] https://github.com/public-ui/kolibri/pull/7227
- [ ] https://github.com/public-ui/public-ui.github.io/pull/270
- [ ] https://github.com/public-ui/kolibri/pull/7237

Wird in separatem Issue erledigt:

- https://github.com/public-ui/kolibri/issues/7118

Zu klären:

- [x] Werden ariaControls und ariaExpanded von Button entfernt? Diese sind als deprecated markiert, es ist aber nicht mehr nachvollziehbar wieso --> Besprochen im Daily vom 2024-12-20: Properties werden nicht entfernt, Deprecated-Annotations entfernen wir.

### PR DESCRIPTION
**Pull request by sdvg**
Refs: #7015

The A11y and PO reviews will only take place after all other DoD steps have been completed by the Developer:
- [x] Meaningful pull request title for the release notes
- [x] Pull request is linked to an issue and all changes relate to the issue
- [x] Tests to protect this code implemented (if applicable)
- [x] Manual test performed successfully (if applicable)
- [x] Documentation or migration has been updated (if applicable)

### CODE REVIEW

File: docs/BREAKING_CHANGES.v3.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ For more information, see the [KoliBri Maintenance and Support Strategy](https:/
 - Visually, the tooltip has been replaced by a simple label shown in parentheses after the abbreviation.
 - The property `_tooltipAlign` has been removed.
 
+## All Input Components
+
+- The property `_alert` has been removed. It's not being handled automatically based on `_msg` and the touched state. See #6138.
```
Comment (laske185) [inline docs/BREAKING_CHANGES.v3.md]: Es wird doch jetzt automatisch durch _msg und _touched gesetzt, oder?

File: packages/components/src/schema/props/alert.ts
Comment (laske185) [inline packages/components/src/schema/props/alert.ts:1]: Kann das jetzt nicht komplett weg?

**Review by laske185 — CHANGES_REQUESTED** [review]

File: docs/BREAKING_CHANGES.v3.md (hunk 1 above)
Comment (sdvg) [reply docs/BREAKING_CHANGES.v3.md]: Typo, es sollte `now` statt `not` heißen

File: packages/components/src/schema/props/alert.ts
Comment (sdvg) [reply packages/components/src/schema/props/alert.ts:1]: Es wird weiterhin von FormFieldMsgFc/AlertFc genutzt.

Comment (laske185) [reply packages/components/src/schema/props/alert.ts:1]: Ok

**Review by laske185 — APPROVED** [review]

**Pull request merged**